Include/internal/pycore_uops.h

```diff
@@ -21,6 +21,9 @@ typedef struct {
 
 typedef struct {
     _PyExecutorObject base;
+    // Auxiliary arrays, allocated after trace[base.ob_size]
+    uint16_t *counters;  // An array of counters
+    _PyExecutorObject **executors;  // An array of executors
```

Comment:
It's possible to compute where these pointers point using the object start and the size, but it's a bit awkward, and the space savings (16 bytes per executor) don't feel attractive.